R/utility.R

```diff
@@ -249,6 +249,65 @@ validateClusterConfig <- function(clusterFilePath){
   TRUE
 }
 
+#' Validating cluster configuration files below doAzureParallel version 0.3.2
+validateDeprecatedClusterConfig <- function(clusterFilePath){
```

Comment:
Going forward we may support multiple of these. I suggest naming it in a version specific way:
validateDepreciatedClusterConfig_0.3.2